src/pybind/mgr/dashboard/frontend/src/app/app-routing.module.ts

```diff
@@ -429,6 +430,11 @@ const routes: Routes = [
                 data: { breadcrumbs: ActionLabels.EDIT }
               }
             ]
+          },
+          {
+            path: 'smb',
+            component: SmbClusterListComponent,
+            data: { breadcrumbs: 'File/SMB' }
```

Comment:
Just a NIT, add it as `children:`

```suggestion
          },
          {
            path: 'smb',
            data: {
              breadcrumbs: 'File/SMB',
            },
            children: [
              { path: '', component: SmbListComponent }
            ]
           }
```

Reply:
Yeah, I was supposed to push these in my form PR. 
But i think i can cover here itself.